src/main/java/io/fusionauth/load/FusionAuthLoginWorker.java

```diff
@@ -29,28 +29,29 @@
  *
  * @author Daniel DeGroff
  */
-public class FusionAuthLoginWorker extends BaseWorker {
-  private final UUID applicationId;
-
-  private final FusionAuthClient client;
+public class FusionAuthLoginWorker extends FusionAuthBaseWorker {
+  private final UUID configuredApplicationId;
 
   private final int loginLowerBound;
 
   private final int loginUpperBound;
 
   public FusionAuthLoginWorker(FusionAuthClient client, Configuration configuration) {
-    super(configuration);
-    this.client = client;
-    this.applicationId = UUID.fromString(configuration.getString("applicationId"));
+    super(client, configuration);
+    this.configuredApplicationId = UUID.fromString(configuration.getString("applicationId"));
     this.loginLowerBound = configuration.getInteger("loginLowerBound", 0);
     this.loginUpperBound = configuration.getInteger("loginUpperBound", 1_000_000);
   }
 
   @Override
   public boolean execute() {
-    int random = new Random().nextInt((loginUpperBound - loginLowerBound) + 1) + loginLowerBound;
-    String email = "load_user_" + random + "@fusionauth.io";
-    ClientResponse<LoginResponse, Errors> result = client.login(new LoginRequest(applicationId, email, Password));
+    int userIndex = new Random().nextInt((loginUpperBound - loginLowerBound) + 1) + loginLowerBound;
+    String email = "load_user_" + userIndex + "@fusionauth.io";
+    setUserIndex(userIndex);
```

Comment:
I'd like to distinguish variable names, but I'm not sure about readability combining the random and set index lines.
```suggestion
    // Pick a random user to log in
    int index = new Random().nextInt((loginUpperBound - loginLowerBound) + 1) + loginLowerBound;
    setUserIndex(index);
    String email = "load_user_" + userIndex + "@fusionauth.io";
```